Mathlib/CategoryTheory/Galois/Topology.lean

```diff
@@ -17,7 +17,7 @@ embedding of `Aut F` into `∀ X, Aut (F.obj X)` where
 
 ## References
 
-- Stacks Project: Tag 0BMQ
+- [Stacks 0BMQ](https://stacks.math.columbia.edu/tag/0BMQ)
```

Comment:
Clearly out of scope for this PR, but I wonder if we should have a mechanism to assign stacks tags to *section* as well.